models/topics.js

```diff
@@ -21,19 +21,29 @@ exports.selectArticles = (topic, sort_by='created_at', order='desc') => {
     LEFT JOIN comments ON comments.article_id = articles.article_id`
 
     if (!validSortBys.includes(sort_by)) {
-       return Promise.reject({status: 404, msg: 'sort_by column not found'})
+       return Promise.reject({status: 400, msg: 'sort_by column not found'})
     }
 
     if (topic !== undefined) {
         queryValues.push(topic)
         queryStr += ` WHERE topic = $3`
     }
 
+    if (!['asc', 'desc'].includes(order.toLowerCase())) {
+        return Promise.reject({status: 400, msg: 'invalid order query'})
+    }
+
     queryStr += ` GROUP BY comments.article_id, articles.author, articles.title, articles.article_id, articles.topic,
     articles.created_at, articles.votes ORDER BY $1, $2;`
```

Comment:
Can I check if this is working correctly?
We can't use the $ syntax for a parameterised query for column names or SQL keywords - only values to be found inside the tables

Reply:
Just fixed this